docs/source/faq.rst

```diff
@@ -205,7 +205,36 @@ Getting info about environments
 
 SEE ALSO: the :doc:`/using/envs` section of our documentation site. 
 
-2. How can I list all installed packages in a specific environment, for example, ``myenv``?
+2. How can I show the name of the active environment in my command prompt?
+
+   The anaconda (or miniconda) installer asks whether it should always add the name 
+   of the active environment to the command prompt by inserting it into the environment 
+   variable called PATH in the Bash shell. The default is no.
```

Comment:
This is wrong? Setting the PATH is completely separate from the changeps1 option, which the installer doesn't do anything with. And setting the PATH defaults to no only on Linux.